tools/xmlGenerator/domainGeneratorConnector.cpp

```diff
@@ -0,0 +1,204 @@
+#include <ParameterMgrFullConnector.h>
+#include <Tokenizer.h>
+
+#include <iostream>
+#include <sstream>
+#include <memory>
+#include <string>
+#include <stdexcept>
+
+using std::string;
+using std::runtime_error;
```

Comment:
Add a custom exception class to differentiate Parameter Framework errors from other ones.